features/plugins.feature

```diff
@@ -65,9 +66,15 @@ Feature: Functionality of Importer and Exporter Plugins
             | json        | <pyproject.toml version> | plugins | exporter | json     |
             | txt         | <pyproject.toml version> | plugins | exporter | text     |
 
+    @skip_only_with_external_plugins
+    Scenario Outline: Custom JSON Import
+        Given we use the config "empty_folder.yaml"
+        When we run "jrnl --import ./features/data/simple_import.json"
```

Comment:
No need to fix this here, but when we move this test to Pytest BDD, we should pipe this json in so it sits in the feature file (instead of having a separate file to manage).